index.js

```diff
@@ -86,72 +84,57 @@ Collector.prototype._notifyIfFlushed = function(callback, err) {
     setImmediate(callback, err);
 };
 
-Collector.prototype._flushSeries = function(seriesName, points, callback) {
+Collector.prototype._flushPoints = function(points, callback) {
     if (!points || points.length === 0) {
         return;
     }
     var self = this;
 
     // only send N points at a time to avoid making requests too large
-    var spliceIndex;
-    if (self._client.protocol == 'udp:' || self._client.protocol == 'line') {
-      spliceIndex = self.computePointCountToSend(points.map(function (point) {
-        return JSON.stringify(point).length;
-      }), MTU_SIZE);
-    } else {
-      spliceIndex = 50;
-    }
+    var spliceIndex = 50
```

Comment:
The batch size was only computed when using UDP, so I removed it since we're only using TCP now.

I think we should use `computePointCountToSend` for UDP too.